modules/win-autoscale/watchmaker-win-autoscale.cfn.json

```diff
@@ -780,6 +836,53 @@
                                     ]
                                 }
                             },
+                            "c:\\cfn\\scripts\\AmazonCloudWatchAgent.zip": {
+                                "source": {
+                                    "Fn::If": [
+                                        "InstallCloudWatchAgent",
+                                        {
+                                            "Ref": "CloudWatchAgentUrl"
+                                        },
+                                        ""
+                                    ]
+                                }
+                            },
+                            "c:\\cfn\\scripts\\AmazonCloudWatchAgent\\aws-cloudwatch-agent-config.json": {
+                                "content": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "{",
+                                            "    \"logs\": {\n",
+                                            "        \"logs_collected\": {\n",
+                                            "            \"files\": {\n",
+                                            "                \"collect_list\": [\n",
+                                            "                    {\n",
+                                            "                        \"file_path\": \"c:\\\\ProgramData\\\\Amazon\\\\AmazonCloudWatchAgent\\\\Logs\\\\amazon-cloudwatch-agent.log\",\n",
+                                            {
+                                                "Fn::Sub": "         \"log_group_name\": \"${WatchmakerLaunchConfigLogGroup}\",\n"
+                                            },
+                                            "                        \"log_stream_name\": \"cloudwatch_agent_logs_{instance_id}\",\n",
+                                            "                        \"timestamp_format\": \"%H:%M:%S %y %b %-d\"\n",
+                                            "                    },\n",
+                                            "                    {\n",
+                                            "                        \"file_path\": \"c:\\\\cfn\\\\log\\\\cfn-init.log\",\n",
+                                            {
+                                                "Fn::Sub": "         \"log_group_name\": \"${WatchmakerLaunchConfigLogGroup}\",\n"
+                                            },
+                                            "                        \"log_stream_name\": \"cloudformation_init_logs_{instance_id}\",\n",
+                                            "                        \"timestamp_format\": \"%H:%M:%S %y %b %-d\"\n",
+                                            "                    }\n",
```

Comment:
Let's add the watchmaker and salt debug logs here. C:\Watchmaker\Logs\watchmaker.log and C:\Watchmaker\Logs\salt_call_debug.log